client/state/automated-transfer/selectors.js

```diff
@@ -48,3 +54,15 @@ export const getEligibility = compose(
 	getEligibilityData,
 	getAutomatedTransfer,
 );
+
+/**
+ * Checks if the site is currently transferring
+ *
+ * @param {Object} state global app state
+ * @param {number} siteId requested site for tranfer info
+ * @returns {bool} transferring check
+ */
+export const isAutomatedTransferTransferring = ( state, siteId ) => includes(
```

Comment:
@aduth this is nice for naming but bad for flat selectors. care to share your thoughts?

do you think it's still possible to group selectors under object properties when they _are_ functionally related?

```js
automatedTransfer.isTransferring( state, siteId )
```